jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -138,22 +138,25 @@ use the `--split-by` flag with the `filesize` split type.
 
     circleci tests glob "**/*.go" | circleci tests split --split-by=filesize
 
-#### Splitting by Timings
+#### Splitting by Timings Data
 
-CircleCI automatically saves timing data from previous successful runs to a default directory.
-This directory is specified by the [built-in environment variable]({{ site.baseurl }}/2.0/env-vars/#circleci-built-in-environment-variables) `CIRCLE_INTERNAL_TASK_DATA`.
-Ensure you are using the [`store_test_results` key]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results)
-to save your timing data,
-or there will be no historical data available.
+On each successful run of a test suite,
+CircleCI automatically saves timings data to a directory
```

Comment:
It's not really automatic since you need to specify it in the config.

Reply:
Whoops! Holdover from the `CIRCLECI_INTERNAL_TASK_DATA` days. :)